app.js

```diff
@@ -0,0 +1,67 @@
+// A very basic web server in node.js
+// Stolen from: Node.js for Front-End Developers by Garann Means (p. 9-10)
```

Comment:
You clearly shown an ability to go beyond the course by learning node.js - kudos to you. For feature testing, I recommend you explore Mocha, or Jest: here's some links you may find handy:

https://jestjs.io/
https://mochajs.org/
https://karma-runner.github.io/1.0/index.html

You can consider using React.js, although I think the testing frameworks above also work with node :)